### PR TITLE
Adding democracy and deps

### PR DESCRIPTION
**Pull request by rhuttman**

### CODE REVIEW

**Comment by irsal** [comment]
To save us time in the future, can you add an item to the CHANGELOG and state that this wil/not bump authoring version pls

File: runtime/neumann/src/lib.rs [hunk 1]
```diff
@@ -176,10 +176,10 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: create_runtime_str!("neumann"),
 	impl_name: create_runtime_str!("neumann"),
 	authoring_version: 1,
-	spec_version: 276,
+	spec_version: 277,
 	impl_version: 1,
 	apis: RUNTIME_API_VERSIONS,
-	transaction_version: 4,
+	transaction_version: 5,
```
Comment (irsal) [inline runtime/neumann/src/lib.rs:182]: oh, beat me to it

**Review by irsal — APPROVED** [review]
sorry i meant tx version, you got it covered

**Review by justinzhou93 — APPROVED** [review]

File: node/src/chain_spec/turing.rs [hunk 2]
```diff
@@ -129,6 +135,28 @@ pub fn turing_latest_latest() -> ChainSpec {
 				initial_allocation,
 				DEFAULT_PARA_ID.into(),
 				vec![],
+				vec![
```
Comment (irsal) [inline node/src/chain_spec/turing.rs:138]: is this the technical?

Comment (irsal) [reply node/src/chain_spec/turing.rs:138]: general

File: node/src/chain_spec/turing.rs [hunk 3]
```diff
@@ -129,6 +135,28 @@ pub fn turing_latest_latest() -> ChainSpec {
...
+					// 67nmVh57G9yo7sqiGLjgNNqtUd7H2CSESTyQgp5272aMibwS
+					hex!["488ced7d199b4386081a52505962128da5a3f54f4665db3d78b6e9f9e89eea4d"].into(),
+					// 67kgfmY6zpw1PRYpj3D5RtkzVZnVvn49XHGyR4v9MEsRRyet
+					hex!["46f630b3f79c588100dc0f69845633a830e01ea09eed4f1d01314a9bf33b9c16"].into(),
+					// 67D6ecyNhnAzZqgRbxr3MdGnxB9Bw8VadMhjpLAYB3wf5Pq6
+					hex!["2edf0fd8948ea642f135b314b1358c77ec6d0a4af83220b6ea18136e5ce36277"].into(),
+					// 6AMsXyV1CYc3LMTk155JTDGEzbgVPvsX9aXp7VXz9heC3iuP
+					hex!["ba44d2c00d9528c2d1fc51cef8ce8b9c3939928ecda8f404cdc46e3a2c090627"].into(),
+				],
+				vec![
```
Comment (irsal) [inline node/src/chain_spec/turing.rs:148]: technical

**Review by irsal — APPROVED** [review]

**Review by okunega1 — COMMENTED** [review]
Review for merger

**Pull request merged**